gettsim/social_insurance_contributions/eink_grenzen.py

```diff
@@ -124,12 +125,59 @@ def midi_job_faktor_f(
     return out
 
 
+@add_rounding_spec(params_key="soz_vers_beitr")
+def midi_job_faktor_f_ab_2022(
+    soz_vers_beitr_params: dict,
+) -> float:
+    """Faktor F which is needed for the calculation of Bemessungsentgelt
+    (beitragspflichtige Einnahme) of midi jobs. It is calculated as the ratio of 28 %
+    divided by the total social security contribution rate
+    (Gesamtsozialversicherungsbeitragssatz).
```

Comment:
Refer to time of introduction of this function (since October 2022). Also do so for previous function. Please also check for all other functions that were "split up in time" in this PR.